### PR TITLE
Correcting placeholder content

### PR DESCRIPTION
**Pull request by sydinh**

### CODE REVIEW

File: src/v2/guide/forms.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Bạn có thể sử dụng directive `v-model` để tạo ràng buộc dữ li
 ### Văn bản
 
 ``` html
-<input v-model="message" placeholder="edit me">
+<input v-model="message" placeholder="Nhập văn bản">
```
Comment (phanan) [inline src/v2/guide/forms.md]: Shouldn't it be "Nhập thông điệp" instead?

**Review by phanan — CHANGES_REQUESTED** [review]

File: src/v2/guide/forms.md (hunk 1 above)
Comment (sydinh) [reply src/v2/guide/forms.md]: Someone corrected it. 😄

Comment (sydinh) [reply src/v2/guide/forms.md]: Corrected it. Please, re-check

**Review by phanan — APPROVED** [review]

**Comment by phanan** [comment]
Thanks!

**Pull request merged**